### ISSUE TITLE
Boost versions is archived

### ISSUE THREAD

**Issue by cvvergara** [MarkusJx/install-boost#4]
this action still works, but eventually it will be outdated because this is an archived repository
https://github.com/actions/boost-versions/blob/main/versions-manifest.json

**Comment by cvvergara** [comment]
This 
https://www.boost.org/users/download/
Has a link that says :
>  Prebuilt windows binaries.

That takes you here    
https://sourceforge.net/projects/boost/files/boost-binaries/

maybe you can use that

**Comment by MarkusJx** [comment]
Yeah, the thing is, these prebuilts are windows-only (although unix-like systems do have prebuilt options). Plus, I'd have to test if those versions work with the github runners, so I'll probably try to recreate the actions/boost-versions repo to supply the prebuilts.

Anyways, thank you for your suggestion, I'll consider using those prebuilts if my plan doesn't work :)

**Comment by MarkusJx** [comment]
Okay, so I've started compiling new Boost versions in a [new repo](https://github.com/MarkusJx/prebuilt-boost). You can try it out using the ``v2.beta.1`` branch version tag. That should work. I think.

I'll close this issue once this branch is merged into ``main`` and version ``2.0.0`` is released.

**Comment by cvvergara** [comment]
Thanks will give it a try on my pgRouting dev repo.
Can not use it on the main repo because we have to test from v1.56
